src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -1,19 +1,72 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
-const ProfileHeader = () => (
-  <div className="vads-u-background-color--primary-darkest vads-u-color--white vads-u-margin-bottom--2 vads-u-padding-y--3">
-    <div className="row">
-      <h1 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal">
-        Your Profile
-      </h1>
-      <h2 className="vads-u-font-size--h3">
-        Kimberly Elizabeth Smith Washington
-      </h2>
-      <h3 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal">
-        United States Army Reserve
-      </h3>
-    </div>
-  </div>
-);
+class ProfileHeader extends Component {
+  static defaultProps = {
+    userFullName: {
+      first: '',
+      middle: '',
+      last: '',
+    },
+    // defaulting to Army for now as placeholder
+    latestBranchOfService: 'A',
+  };
 
-export default ProfileHeader;
+  static propTypes = {
+    userFullName: PropTypes.shape({
+      first: PropTypes.string.isRequired,
+      middle: PropTypes.string.isRequired,
+      last: PropTypes.string.isRequired,
+    }).isRequired,
+    // defaulting to Army for now as placeholder
+    latestBranchOfService: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const imagePaths = {
+      A: '/img/vic-army-symbol.png',
+      C: '/img/vic-cg-emblem.png',
+      F: '/img/vic-air-force-coat-of-arms.png',
+      N: '/img/vic-navy-emblem.png',
+      M: '/img/vic-usmc-emblem.png',
+      VASeal: '/img/vic-va-seal.png',
+    };
+
+    const {
+      userFullName: { first, middle, last },
+      latestBranchOfService,
+    } = this.props;
+
+    return (
+      <div className="headerWrapper vads-u-background-color--gray-dark vads-u-color--white vads-u-margin-bottom--2 vads-u-padding-y--3 vads-u-display--flex vads-u-align-items--center vads-u-justify-content--center">
+        <div className="headerContentWrapper usa-grid usa-grid-full vads-u-display--flex vads-u-align-items--center">
+          <div className="vads-u-padding-x--6 usa-width-one-fourth">
+            <img
+              className="profileServiceBadge"
+              src={imagePaths[`${latestBranchOfService}`]}
+              alt="service badge"
+            />
+          </div>
+          <div className="headerNameWrapper vads-u-flex-direction--column">
+            <h1 className="profileHeading vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal">
+              Your Profile
+            </h1>
+            <h2 className="vads-u-font-size--h3">
+              {first.toLowerCase()} {middle.toLowerCase()} {last.toLowerCase()}
```

Comment:
A few thoughts on this:
- let's go ahead and remove the `toLowerCase()` calls. I know that right now that'll mean the name will be in ALL CAPS because that's how it's stored in `profile.userFullName`, but once we start pulling this from `hero.userFullName` it'll be in the proper case.
- we should really create a helper function to mash the three name components into one. that way it'll elegantly handle a missing middle name without adding an extra space between the first and last name